tests/log-in.test.js

```diff
@@ -0,0 +1,43 @@
+import LoginPage from '../pageobjects/login.page'
+
+describe('Banking', () => {
+
+    it(' Should access to homepage with correct username and password ', async () => {
+        await LoginPage.openWeb();
+
+        await LoginPage.logIn('jsmith@demo.io', 'Demo123!');
+
+        await expect(await browser.getUrl()).to.equal('http://localhost:8080/bank/home');
```

Comment:
Consider using wdio specific matchers, as they are more idiomatic, shorter and specifically built for asserting browser or web element's attributes.

Also, prefer using relative URLs instead, as that decouples your test from a specific environment (for example, if you run this test against a remotely hosted version of this application it won't work, as it's tailored for something hosted on your local machine)

Tha'll simplify this to something like:

```javascript
await expect(browser).toHaveUrlContaining('home');
```